bpf/lib/auth.h

```diff
@@ -14,7 +14,7 @@ struct {
 	__type(key, struct auth_key);
 	__type(value, struct auth_info);
 	__uint(pinning, LIBBPF_PIN_BY_NAME);
-	__uint(max_entries, AUTH_MAP_SIZE);
```

Comment:
So, are all these changes to fixed values because those fixed values are now effectively the default rather than an actually fixed value?